fgd/brush/trigger/trigger_catapult.fgd

```diff
@@ -27,6 +27,10 @@
 	lowerthreshold(float) : "Lower Threshold" : 0.15 : "Flung object must be within this percentage value in order to activate fling. Specify a value between [0...1] (default is .15). This is only used if Use Threshold Check is set to yes."
 	upperthreshold(float) : "Upper Threshold" : 0.30 : "Flung object must be within this percentage value in order to activate fling. Specify a value between [0...1] (default is .30). This is only used if Use Threshold Check is set to yes."
 	entryangletolerance(float) : "Entry Angle Tolerance" : 0.0 : "Flung object's velocity must be pointing this much at the target. Specify a value between [-1...1] 1 means exactly, 0 means within 180 degrees -1 means any angle is accepted. This is only used if Use Threshold Check is set to yes."
+	
+	UseImpulses[P2EDU](float) : "Use Impulses" : "0" : "Use impulses to determine launched object velocity."
```

Comment:
I think this one should be a `bool`?

Reply:
You could be right. I only used "float" because the edu FGD itself uses float for it.